.readthedocs.yaml

```diff
@@ -0,0 +1,9 @@
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.12"
+
+sphinx:
+  configuration: conf.py
```

Comment:
Is this file necessary? AFAIK these docs don't end up on readthedocs?

Reply:
![image](https://github.com/editorconfig/specification/assets/10515894/06a55e0c-df53-4348-adb7-47180141bde5)

;)

Reply:
As @cxw42 showed with his screenshot, the specs are hosted on readthedocs

Also see the output of `dig` below. spec.editorconfig.org is a CNAME to editorconfig-specification.readthedocs.io

```
$ dig spec.editorconfig.org
; <<>> DiG 9.18.18-0ubuntu0.22.04.1-Ubuntu <<>> spec.editorconfig.org
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 14302
;; flags: qr rd ad; QUERY: 1, ANSWER: 3, AUTHORITY: 0, ADDITIONAL: 0
;; WARNING: recursion requested but not available

;; QUESTION SECTION:
;spec.editorconfig.org.         IN      A

;; ANSWER SECTION:
spec.editorconfig.org.  0       IN      CNAME   editorconfig-specification.readthedocs.io.
editorconfig-specification.readthedocs.io. 0 IN A 104.17.32.82
editorconfig-specification.readthedocs.io. 0 IN A 104.17.33.82

;; Query time: 429 msec
;; SERVER: 172.21.32.1#53(172.21.32.1) (UDP)
;; WHEN: Thu Feb 01 13:52:49 CET 2024
;; MSG SIZE  rcvd: 188
```